### PR TITLE
Rating and toast

### PR DESCRIPTION
**Pull request by romabulani**
- Rating bar with 5 Stars 
- 4 types of Toast (success, info, warning and error) with JS
- Added component css file after documentation css for every element.

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Your ui looks very clean and easy to use. Components are also relevant to use case and easy to place.

File: Components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.toast{
+    border-radius: 5px;
+    bottom: 1rem;
+    display: none;
+    left: 1rem;
+    padding: 0.625rem 2rem;
+    position: absolute;
+    position: absolute;
```
Comment (manish591) [inline Components/toast/toast.css]: You have written it twice.

**Review by manish591 — COMMENTED** [review]
Looks good.

File: Components/toast/toast.css (hunk 1 above)
Comment (romabulani) [reply Components/toast/toast.css]: @manish591 Thanks for pointing it out, removed the redundant property

**Comment by romabulani** [comment]
> Your ui looks very clean and easy to use. Components are also relevant to use case and easy to place.

Thankyou :)

**Review by manish591 — APPROVED** [review]

**Pull request merged**